### PR TITLE
docs: bump Mayastor and OpenEBS versions

### PR DESCRIPTION
**Pull request by balaharish7**
Changed the OpenEBS version 4.0.0 to 4.0.1 & Mayastor version 2.6.0 to 2.6.1 in all commands and in respective contents.

### CODE REVIEW

File: docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/legacy-upgrade.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ keywords:
  - Upgrading OpenEBS
  - OpenEBS upgrade
  - Supported upgrade paths
-description: Upgrade to the latest OpenEBS 2.6.0 version is supported only from v1.0.0 and later.
+description: Upgrade to the latest OpenEBS 2.6.1 version is supported only from v1.0.0 and later.
```
Comment (niladrih) [inline docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/legacy-upgrade.md:8]: This is not needed.

File: docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/upgrade-re.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ keywords:
  - Upgrading OpenEBS
  - OpenEBS upgrade
  - Supported upgrade paths
-description: Upgrade to the latest OpenEBS 2.6.0 version is supported only from v1.0.0 and later.
+description: Upgrade to the latest OpenEBS 2.6.1 version is supported only from v1.0.0 and later.
```
Comment (niladrih) [inline docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/upgrade-re.md:8]: Not needed.

File: docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/legacy-upgrade.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ keywords:
  - Upgrading OpenEBS
  - OpenEBS upgrade
  - Supported upgrade paths
-description: Upgrade to the latest OpenEBS 2.6.0 version is supported only from v1.0.0 and later.
+description: Upgrade to the latest OpenEBS 2.6.1 version is supported only from v1.0.0 and later.
```
Comment (niladrih) [inline docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/legacy-upgrade.md:8]: Not needed.

File: docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/upgrade-re.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ keywords:
  - Upgrading OpenEBS
  - OpenEBS upgrade
  - Supported upgrade paths
-description: Upgrade to the latest OpenEBS 2.6.0 version is supported only from v1.0.0 and later.
+description: Upgrade to the latest OpenEBS 2.6.1 version is supported only from v1.0.0 and later.
```
Comment (niladrih) [inline docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/upgrade-re.md:8]: Not needed.

**Review by avishnu — APPROVED** [review]
lgtm

**Review by niladrih — APPROVED** [review]
LGTM. But I think we should hold off on merging this before we're fairly close to releasing 4.0.1.

**Pull request merged**